app/views/validators/show.html.erb

```diff
@@ -123,7 +123,10 @@
               @validator.commission.to_i == 100
           %>
           <td class="<%= comm_class %>">
-            <%= @validator.commission rescue 'N/A' %> &percnt;
+            <%= @validator.commission rescue 'N/A' %>&percnt;
+            <%= link_to '(See commission changes)',
+                        commission_histories_path(network: params[:network], validator_id: @validator.id),
+                        class: 'small' %>
```

Comment:
Please make this link visible only when there are commission changes to show. It is bad UX for the user to click a link and then have nothing to show them.